src/elements/DoubleSlider/DoubleSlider.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="DoubleSlider" :style="cssVars">
```

Comment:
Would much prefer you use the class 'vocab slider'.

Reply:
should I change the component name to Slider as well. it seems kind of strange that the component class name would be slider and not doubleSlider

Reply:
Ideally the name would have the `-Field` suffix (as `InputField`, `ChoiceField`, `SwitchField` etc.)